app/assets/javascripts/users.coffee

```diff
@@ -0,0 +1,10 @@
+$ ->
+  $(document.body).on "click", "[data-action]", (e) ->
```

Comment:
We like often to work on principle of "least-astonishment".  Straightforward clear code.  I'll suggest the following...
```
<a href="/users/5/trim" class="update-closest-status">Trim</a>
```

OR
```
<a href="/users/5/trim" data-update-closest-tr=".status">Trim</a>
```
which would be far clearer to the next developer when reading your code.  Both of those solutions can be implemented with a global selector similar to what you have here.